extensions-contrib/materialized-view-selection/src/main/java/org/apache/druid/query/materializedview/DerivativeDataSourceManager.java

```diff
@@ -65,7 +65,7 @@
  * Read and store derivatives information from dataSource table frequently.
  * When optimize query, DerivativesManager offers the information about derivatives.
  */
-@ManageLifecycleLast
+@ManageLifecycleAnnouncements
```

Comment:
Why is this in `@ManageLifecycleAnnouncements`? It doesn't seem to be doing announcements?

Reply:
Good question, I'll see if I can dig up why it was marked `@ManageLifecycleLast` prior to this PR...

Reply:
There is no reason in the initial PR, it was already in the first commit and not an explicit response to review afaict. Looking over what it is doing, it doesn't seem like it would need to be in the last stage, I will move it to just `@ManageLifecycle`

Reply:
Ok, moved it to `Lifecycle.Stage.NORMAL`, thanks for review!